site/docs/tutorial/android-app.md

```diff
@@ -3,140 +3,149 @@ layout: documentation
+                        ├── Greeter.java
+                        ├── MainActivity.java
+                        └── res
+                            ├── layout
+                            │   └── activity_main.xml
+                            └── values
+                                ├── colors.xml
+                                └── strings.xml
 ```
 
-The `git clone` command creates a directory named `$HOME/examples/`. This
-directory contains several sample projects for Bazel. The project files for this
-tutorial are in `$HOME/examples/tutorial/android`.
+The key files and directories are:
 
-## Set up a workspace
+| Name                    | Location                                                                                      |
+| Android manifest files  | `src/main/java/AndroidManifest.xml` and `src/main/java/com/example/bazel/AndroidManifest.xml` |
+| Activity source files   | `src/main/java/com/example/bazel/MainActivity.java` and `Greeter.java`                        |
+| Resource file directory | `src/main/java/com/example/bazel/res/`                                                        |
 
-A [workspace](../build-ref.html#workspaces) is a directory that contains the
-source files for one or more software projects, as well as a `WORKSPACE` file
-and `BUILD` files that contain the instructions that Bazel uses to build
-the software. The workspace may also contain symbolic links to output
-directories.
+Note that you're just looking at these files now to become familiar with the
+structure of the app. You don't have to edit any of the source files to complete
+this tutorial.
 
-A workspace directory can be located anywhere on your filesystem and is denoted
-by the presence of the `WORKSPACE` file at its root. In this tutorial, your
-workspace directory is `$HOME/examples/tutorial/`, which contains the sample
-project files you cloned from the GitHub repo in the previous step.
+## Initialize the Bazel workspace
 
-Note that Bazel itself doesn't make any requirements about how you organize
-source files in your workspace. The sample source files in this tutorial are
-organized according to conventions for the target platform.
+A [workspace](../build-ref.html#workspace) is a directory that contains the
+source files for one or more software projects, and a `WORKSPACE` file in the
+top level of the directory.
 
-For your convenience, set the `$WORKSPACE` environment variable now to refer to
-your workspace directory. At the command line, enter:
+The `WORKSPACE` file may be empty or it may contain references to [external
+dependencies](../external.html) required to build the app. Run the following
+command to create an empty `WORKSPACE` file:
 
-```bash
-export WORKSPACE=$HOME/examples/tutorial
-```
+| Linux, macOS             | `$ touch WORKSPACE`                   |
+| Windows (Command Prompt) | `$ type nul > WORKSPACE`              |
+| Windows (PowerShell)     | `$ New-Item WORKSPACE -ItemType file` |
```

Comment:
I recommend omitting the `$` prompt indicator (for all platforms, for sake of consistency).
AFAIK it's not customary on Windows and may be confusing and considered part of the command.

Reply:
done, removed all `$` prompts.